DokuwikiXMLExport.php

```diff
@@ -238,10 +254,9 @@ private function getPageId($pageId)
      */
     private function getAttributesCategory($pageId)
     {
-        $ordernumber = $this->getPageId($pageId);
-
-        $attribute = str_replace('_', ' ', $ordernumber);
-        $attribute = str_replace(':', '_', $attribute);
+        $attribute = str_replace(self::CATEGORY_DELIMITER, ' ', $pageId); // '_' -> ' '
```

Comment:
Always good to provide an explanatory comment for these kind of things. :+1:

Reply:
In the PHPDoc I have added 1 more example and I edited the comments next to the code line.